components/Team/Team.tsx

```diff
@@ -1,23 +1,29 @@
 import { Box } from "@mui/material";
-import React from "react";
 import { LinearGradient } from "../LinearGradient";
 import { SectionTitle } from "../SectionTitle";
 import Bridge from "./Bridge";
 import TeamCards from "./TeamCards";
 
-export default function Team () {
+export default function Team() {
 
+  const sx = {
+    width: "100%",
+    height: "auto",
+    backgroundColor: "#1B1B1B",
+    display: "flex",
+    flexDirection: "column",
+    color: "white",
+    justifyContent: "center", alignItems: "center"
+  };
 
-    const sx = { width: '100%', height: 'auto', backgroundColor: '#1B1B1B' , display:'flex' , flexDirection: 'column', color: 'white' , justifyContent: "center", alignItems:'center', };
-
-    return (
-			<>
-            <Box sx={sx}>
-				<SectionTitle sx={{marginTop: '40px'}}mb={2}> Let's meet us</SectionTitle>
```

Comment:
avoid such a spaghetti guys :P set up auto-formatting in your IDE and problem will be solved